cypress/support/step-definitions/textbox-steps.js

```diff
@@ -229,3 +229,20 @@ Then('Multiple label Align on preview is {string}', (direction) => {
   labelByPosition(FIRST_ELEMENT).should($element => expect($element).to.have.css(TEXT_ALIGN, `${direction}`));
   labelByPosition(SECOND_ELEMENT).should($element => expect($element).to.have.css(TEXT_ALIGN, `${direction}`));
 });
+
+Then('I click on icon inside of Textbox', () => {
+  icon().click();
+});
+
+Then('icon on preview is {string} and is visible', (iconName) => {
+  icon().should('have.attr', 'data-element', iconName)
```

Comment:
we should use new locator to find the icon inside the Textbox input

Reply:
fixed